batch/batch/front_end/front_end.py

```diff
@@ -1626,10 +1629,52 @@ def plot_job_durations(container_statuses: dict, batch_id: int, job_id: int):
     return json.dumps(fig, cls=plotly.utils.PlotlyJSONEncoder)
 
 
+
+
 def plot_resource_usage(
         resource_usage: Optional[Dict[str, Optional[pd.DataFrame]]]
 ) -> Optional[str]:  # pylint: disable=unused-argument
-    return None
+    fig = make_subplots(
+        rows=2, cols=1,
+        subplot_titles=("Memory", "CPU Usage")
+    )
+    colors = {'input': 'red', 'main': 'green', 'output': 'blue'}
+
+    if resource_usage is None:
```

Comment:
Let's move this check for `resource_usage` is None to the very beginning of the function.

Reply:
Added change